model/post_test.go

```diff
@@ -121,15 +105,11 @@ func TestPostIsSystemMessage(t *testing.T) {
 	post1 := Post{Message: "test_1"}
 	post1.PreSave()
 
-	if post1.IsSystemMessage() {
-		t.Fatalf("TestPostIsSystemMessage failed, expected post1.IsSystemMessage() to be false")
-	}
+	require.False(t, post1.IsSystemMessage(), "post1.IsSystemMessage() should be false")
```

Comment:
nit: as the check already states that the value should be false, the message string is redundant and can be removed